.pa11yci

```diff
@@ -124,6 +124,11 @@
 		"http://127.0.0.1:8080/patterns/elements-input-input-week/elements-input-input-week.rendered.html",
 		"http://127.0.0.1:8080/patterns/elements-link-links/elements-link-links.rendered.html",
 		"http://127.0.0.1:8080/patterns/elements-link-links-small/elements-link-links-small.rendered.html",
+		"http://127.0.0.1:8080/patterns/elements-loading-indicator-01-loading-indicator-size-XS/elements-loading-indicator-01-loading-indicator-size-XS.rendered.html",
+		"http://127.0.0.1:8080/patterns/elements-loading-indicator-02-loading-indicator-size-s/elements-loading-indicator-02-loading-indicator-size-s.rendered.html",
+		"http://127.0.0.1:8080/patterns/elements-loading-indicator-03-loading-indicator-size-m/elements-loading-indicator-03-loading-indicator-size-m.rendered.html",
+		"http://127.0.0.1:8080/patterns/elements-loading-indicator-04-loading-indicator-size-l/elements-loading-indicator-04-loading-indicator-size-l.rendered.html",
+		"http://127.0.0.1:8080/patterns/elements-loading-indicator-05-loading-indicator-size-XL/elements-loading-indicator-05-loading-indicator-size-XL.rendered.html",
```

Comment:
Please add those to the backstopjs URLs as well.

Reply:
@mfranzke how did you set the reference in the past when the final reference is not deployed yet? According to the other components the reference url is set to sth. like https://db-ui.github.io/core/patterns/.../....rendered.html

Reply:
@annsch pa11y doesn't need a reference, but backstopjs does. In case that backstopjs doesn't find a reference file, it's still okay to provide the URL – you'll get an error on those URLs, and approve them, to add the reference screenshots, so you've prepared your stack for the next test.

Reply:
@mfranzke I added those urls and finally could approve the tests